pom.xml

```diff
@@ -35,6 +35,7 @@
 		<selenide.version>4.8</selenide.version>
 		<guava.version>22.0</guava.version>
 		<cucumber.version>2.1.0</cucumber.version>
+		<cucumber.spring>2.3.1</cucumber.spring>
```

Comment:
We could probably upgrade all Cucumber deps with the `cucumber.version`, hopefully without issues. :)

Reply:
Ok, I was just afraid I would break something, but I think we could try it.

Reply:
There're just 3 releases in between `2.1.0` vs. `2.3.1`. Nothing to worry about 🐉